javascript/pane.js

```diff
@@ -52,13 +43,21 @@ function Pane(options){
   $(document).on('keyup', function(event){
     if (event.which == 27 && isOpen){
       context.hide()
-      options.trigger.focus()
+      return false
     }
   })
 
 
   // HELPER FUNCTIONS
 
+  function paneIsOpen(){
+    return isOpen
+  }
+
+  function paneIsClosed(){
+    return !isOpen
+  }
+
```

Comment:
The naming pattern I've been using for methods that get the state of something is `isSomethingSomething()`. See methods like `isEventOutsidePane`.

Reply:
@njakobsen I'm using the same naming convention, but in the public API in this case.  I only named the private name this way because if I named it "isOpen" it would shadow the "isOpen" variable, and I just made "isClosed" match.  In the public API they are "pane.isOpen()" and "pane.isClosed()" respectively.  If you'd still like me to do something different please advise.  Thank you.

Reply:
I was thinking `isPaneClosed` however now that you mention it, is this method necessary? We already have a property that gives us this information.

Reply:
@njakobsen Necessary would be a strong word to use.  Just for me I prefer the readability of

```javascript
if (pane.isClosed()) {
    // ...
}
```

to 

```javascript
if (!pane.isOpen()) {
    // ...
}
```

It's just a style thing.  Like if you think it's meaningful that Ruby has constructs like `unless`.